browser/components/molecules/Timeline/index.js

```diff
@@ -0,0 +1,119 @@
+import React, { Component, PropTypes } from 'react'
+import moment from 'moment'
+import getPrrrsPerWeek from '../../../actions/getPrrrsPerWeek'
+import Button from '../../atoms/Button'
+import './index.sass'
+import Link from '../../atoms/Link'
+
+export default class Timeline extends Component {
+  static propTypes = {
+    week: PropTypes.instanceOf(moment).isRequired,
+  }
+
+  constructor(props){
+    super(props)
+    getPrrrsPerWeek(props.week)
+  }
+
+  componentWillReceiveProps(nextProps){
+    if (!this.props.week.isSame(nextProps.week)){
+      getPrrrsPerWeek(nextProps.week)
+    }
+  }
+
+  timeFromStartofWeekInSeconds = (inputTime) => {
+    return Math.round((inputTime.valueOf() - this.props.week.clone().valueOf()) / 1000)
+  }
+
+  areaLeftofPoints = (timeInput) => {
+    return Math.round((this.timeFromStartofWeekInSeconds(timeInput) / 604800) * 100)
```

Comment:
please name constants like `604800`. I normally put these at the top of the file like:

```js
const ONE_WEEK_IN_SECONDS = 604800
```